.github/workflows/build.yaml

```diff
@@ -99,12 +116,13 @@ jobs:
         run: |
           make tools
 
-      - name: Run integration tests with Pact
+      - name: Run integration & smoke tests with Pact
         if: matrix.os == 'ubuntu-latest'
         env:
           DEEPROXY_API_URL: ${{secrets.DEEPROXY_API_URL}}
           SNYK_TOKEN: ${{secrets.SNYK_TOKEN }}
           INTEG_TESTS: "true"
+          SMOKE_TESTS: "true"
```

Comment:
enabling smoke tests here, so we catch issues before we merge to main

Reply:
![image](https://github.com/user-attachments/assets/96d9fe5c-a2e1-4022-9b70-fb6a8621ca4f)